CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+FEATURES
+* Added support for Consul Enterprise Admin Partitions and Namespaces.
```

Comment:
Should be `Add support...`

Reply:
Agreed.. Not really too much of a chicken/egg in this case because I already had the PR created.